### PR TITLE
chore: upgrade to yarn berry from yarn 1

### PR DESCRIPTION
**Pull request by BugGambit**
Jira: https://cognitedata.atlassian.net/browse/AUTH-3027

### CODE REVIEW

**Comment by samir-ajdarpasic** [comment]
Don't you need the yarn 4 version in the codebase as we do in the other repos ? Because the `setup-node` action doesn't yet support yarn 4.

File: .yarnrc.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```
Comment (samir-ajdarpasic) [inline .yarnrc.yml:1]: You need to add `yarnPath: .yarn/releases/yarn-4.4.0.cjs`

**Review by samir-ajdarpasic — APPROVED** [review]

**Comment by samir-ajdarpasic** [comment]
@BugGambit Can we also update the github action versions to latest so they also use Node 20 ?

**Comment by BugGambit** [comment]
> @BugGambit Can we also update the github action versions to latest so they also use Node 20 ?

@samir-ajdarpasic that should already be updated in this PR?

**Comment by samir-ajdarpasic** [comment]
> > @BugGambit Can we also update the github action versions to latest so they also use Node 20 ?
> 
> @samir-ajdarpasic that should already be updated in this PR?

Ref: https://github.com/cognitedata/fusion/commit/69cfc9c3898b935d141c06630fff1fb56ad636c4

**Comment by BugGambit** [comment]
> > > @BugGambit Can we also update the github action versions to latest so they also use Node 20 ?
> > 
> > 
> > @samir-ajdarpasic that should already be updated in this PR?
> 
> Ref: [cognitedata/fusion@69cfc9c](https://github.com/cognitedata/fusion/commit/69cfc9c3898b935d141c06630fff1fb56ad636c4)

d747465e96

File: .github/workflows/release.yaml [hunk 2]
```diff
@@ -10,12 +10,12 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
       
       - name: Use Node.js
         uses: actions/setup-node@v3
```
Comment (samir-ajdarpasic) [inline .github/workflows/release.yaml]: @BugGambit Small nit, but this also should be updated to v4

Comment (BugGambit) [reply .github/workflows/release.yaml]: 4a04b9d209

**Comment by BugGambit** [comment]
Blocked by: https://github.com/cognitedata/terraform/pull/10276

**Pull request merged**